SimplenoteUITests/Generic.swift

```diff
@@ -0,0 +1,80 @@
+import XCTest
+
+var stepIndex = 0
+
+func attemptLogOut() -> Bool {
+    let allNotesNavBar = app.navigationBars[uidNavBar_AllNotes]
+    var loggedOut: Bool = false
+
+    if allNotesNavBar.exists {
+        loggedOut = logOut()
+    } else {
+        loggedOut = true
+    }
+
+    return loggedOut
+}
+
+func logOut() -> Bool {
+    app.navigationBars[uidNavBar_AllNotes].buttons[uidButton_Menu].tap()
+    app.tables.staticTexts[uidCell_Settings].tap()
+    app.tables.staticTexts[uidButton_Settings_LogOut].tap()
+    return app.buttons[uidButton_LogIn].waitForExistence(timeout: maxLoadTimeout)
+}
+
+public func trackTest(_ function: String = #function) {
+    print("> Test: \(function)")
+    stepIndex = 1
+}
+
+func logStep(){
```

Comment:
Nitpick for consistent naming with `trackTest` above:

```suggestion
func trackStep() {
```